src/components/SingleDatePickerInput.jsx

```diff
@@ -122,13 +121,11 @@ export default class SingleDatePickerInput extends React.Component {
               'SingleDatePickerInput__clear-date--hide': !displayValue,
               'SingleDatePickerInput__clear-date--hover': isClearDateHovered,
             })}
+            aria-label={phrases.clearDate}
```

Comment:
Yes. I figured we'd do some of the small aria-labeling/add those phrases unrelated to keyboard nav here.